.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
                 "WaitOnNormalExit",
                 "RedirectOutput"
             ],
-            "debugStdLib": true
+            "justMyCode": false
```

Comment:
I'd remove this kind of configuration change, as it isn't strictly part of solving the problem. :)

Reply:
I could put it in a separate PR, but it is literally just because VSCode complains about it.

Reply:
Fair enough, it's certainly not a big enough deal to block a PR about.